### PR TITLE
Pull req - Fixed issue #103 [You cannot sign and use web inspector anymore]

### PR DESCRIPTION
**Pull request by jamesjhedley**
Fixed Packager issue #103 [blackberry/BB10-Webworks-Packager/issues/103]
+unit tests

### CODE REVIEW

**Comment by nukulb** [comment]
CI build is failing

**Comment by nukulb** [comment]
I think jake test is failing

File: test/unit/lib/native-packager.js [hunk 1]
```diff
@@ -125,4 +126,38 @@ describe("Native packager", function () {
...
+        session.storepass = "password";
+        config.buildId = "100";
+        
+        //Set -d param
+        session.debug = "";
+
+        nativePkgr.exec(session, target, config, callback);
+
+        //options file should NOT contain -devMode
+        expect(fs.writeFileSync).toHaveBeenCalledWith(jasmine.any(String), optionsFile);
```
Comment (nukulb) [inline test/unit/lib/native-packager.js:166]: your comment say "options file should NOT contain -devmode"
and the 
expect simply cares that it is being called. Is there a way you can improve this expect?

Comment (nukulb) [inline test/unit/lib/native-packager.js:166]: ahh I think I get it now, ignore the last comment

**Comment by jamesjhedley** [comment]
I have fixed the branch. The issue was a path issue on Mac and some wacky behavior with path.resolve.

**Comment by tracyli** [comment]
tested in windows and mac with http://mac-ci:9000/job/BB10-Webworks-Packager-next-fix-103/2/ and sign and web inspector work great as expected.

**Comment by gtanner** [comment]
w00t!

**Comment by nukulb** [comment]
code review complete

**Comment by nukulb** [comment]
merged manually

**Pull request closed without merging**